porcupine/plugins/find.py

```diff
@@ -163,6 +163,7 @@ def show(self) -> None:
             self.find_entry.focus_set()
 
         self.highlight_all_matches()
+        self.update_idletasks()
```

Comment:
```suggestion

        # weird hack to prevent rendering issue on mac
        # https://stackoverflow.com/questions/55366795/does-anyone-know-why-my-tkinter-buttons-arent-rendering
        self.update_idletasks()  
```